src/promptflow-parallel/pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[tool.poetry]
+name = "promptflow-parallel"
+version = "1.10.0.dev0"
```

Comment:
may start with some 0.x version, no need to align with promptlfow package?

Reply:
changed to 0.1.0.dev0